contracts/contracts/MerkleDrop.sol

```diff
@@ -0,0 +1,40 @@
+pragma solidity ^0.5.8;
+
+
+contract MerkleDrop {
+
+    bytes32 public root;
+
+    constructor(bytes32 _root) public {
+        root = _root;
+    }
+
+    function withdraw(address recipient, uint value, bytes32[] memory proof) public {
```

Comment:
we could have 
`withdraw(value, proof)` and `withdrawFor(recipient, value, proof)`
for easier usage
What do you think?

Reply:
Sure.